virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -955,7 +954,7 @@ You must manually disconnect the console connection if you open a new console se
 [id="virtualmachine-details-snapshots_virt-web-console-overview"]
 ==== Snapshots tab
 
-You create snapshots and restore virtual machines from snapshots on the *Snapshots* tab.
+You can create a snapshot, create a copy of a virtual machine from a snapshot, restore a snapshot as a new PVC, edit labels or annotations, and edit or delete volume snapshots on the *Snapshots* tab.
```

Comment:
What does is mean to restore a snapshot as a new PVC? I dont believe this is an option, there is an option of restore a vm from a snapshot.

Reply:
These selections are shown in the new screenshot provided by Yifat in the design doc updated on Feb 14: https://docs.google.com/document/d/1ovuHz7kJ0T9ZaGnrVOj3zgeJSMSnk2OoIgZhhVwlC_k/edit#heading=h.5gbmlqdgw69f

Reply:
Agree with Shelly, we can't restore a snapshot as a new PVC. This looks more correct:

`You can create a snapshot, create a copy of a virtual machine from a snapshot, restore a snapshot, edit labels or annotations, and edit or delete volume snapshots on the *Snapshots* tab.`